### ISSUE TITLE
marking up quotation marks in 4.0.1?

### ISSUE THREAD

**Issue by AnnaParla** [ufal/ParlaMint-UA#79]
The current README file says: Quotation marks have been left in the text and are not explicitly marked up. I wonder if it is relatively easy to mark them up in the future, which will be very useful for the differentiation between, for instance, ЄС (abbreviated European Union in uk) and "ЄС" (abbreviated political party and faction European Solidarity in uk).

**Comment by matyaskopp** [comment]
You can always search for ЄС surrounded by specific tokens: https://www.clarin.si/ske/#concordance?corpname=parlamint40_ua&tab=advanced&queryselector=cql&attrs=word&viewmode=sen&attr_allpos=all&refs_up=0&shorten_refs=1&glue=1&gdexcnt=300&show_gdex_scores=0&itemsPerPage=20&structs=s%2Cg&refs=%3Dspeech.speaker_id%2C%3Dspeech.date&default_attr=lemma&cql=%5Bword%3D%22%5C%22%22%5D%20%5Bword%3D%22%D0%84%D0%A1%22%5D%20%5Bword%3D%22%5C%22%22%5D&showresults=1&showTBL=0&tbl_template=&gdexconf=&cb=sword%3D%22%5C%5C%22%22%23sword%3D%22%D0%84%D0%A1%22%23sword%3D%22%5C%5C%22%22&f_tab=basic&f_showrelfrq=1&f_showperc=0&f_showreldens=0&f_showreltt=0&c_customrange=0&operations=%5B%7B%22name%22%3A%22cql%22%2C%22arg%22%3A%22%5Bword%3D%5C%22%5C%5C%5C%22%5C%22%5D%20%5Bword%3D%5C%22%D0%84%D0%A1%5C%22%5D%20%5Bword%3D%5C%22%5C%5C%5C%22%5C%22%5D%22%2C%22query%22%3A%7B%22queryselector%22%3A%22cqlrow%22%2C%22cql%22%3A%22%5Bword%3D%5C%22%5C%5C%5C%22%5C%22%5D%20%5Bword%3D%5C%22%D0%84%D0%A1%5C%22%5D%20%5Bword%3D%5C%22%5C%5C%5C%22%5C%22%5D%22%2C%22default_attr%22%3A%22lemma%22%7D%2C%22id%22%3A3084%7D%5D

![image](https://github.com/ufal/ParlaMint-UA/assets/5867995/79216daf-9128-4fbd-b0dc-5c2a415d4145)

**Comment by AnnaParla** [comment]
Yes, I use this combination when needed. But more commonly I need to search lemmas without the quotation marks and the result of using [word!="\""] [lemma="ЄС"] [word!="\""] is below. 

![image](https://github.com/ufal/ParlaMint-UA/assets/115141649/b81507c6-5df5-46a8-a56a-88b575553845)

It incudes one word or punctuation mark before and after the search node, which is problematic if I need? for instance, stats for collocations with ЄС -1L.  
![image](https://github.com/ufal/ParlaMint-UA/assets/115141649/7b9c5fb6-7fc7-425e-aab1-bc6156b5d063)

Then I end up with what precedes the whole [word!="\""] [lemma="ЄС"] [word!="\""] and not ЄС itself. 

![image](https://github.com/ufal/ParlaMint-UA/assets/115141649/a0cea16e-90c1-48c9-aa75-7f52ffd53d6a)

**Comment by AnnaParla** [comment]
And in this particular case I cannot filter by name entity, because both ЄС and "ЄС" are ORG, provided they are marked correctly, of course. ;(
